llama_hub/feishu_docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Feishu Doc Loader
+
+This loader takes in IDs of Feishu Docs and parses their text into `documents`. You can extract a Feishu Doc's ID directly from its URL. For example, the ID of `https://test-csl481dfkgqf.feishu.cn/docx/HIH2dHv21ox9kVxjRuwc1W0jnkf` is `HIH2dHv21ox9kVxjRuwc1W0jnkf`. As a prerequisite, you will need to register with Feishu and build an custom app. See [here](https://open.feishu.cn/document/home/introduction-to-custom-app-development/self-built-application-development-process) for instructions.
+
+## Usage
+To use this loader, you simply need to pass in an array of Feishu Doc IDs. The default API endpoints are for Feishu, in order to switch to Lark, we should use `set_lark_domain`.
+
+
+```python
+from llama_index import download_loader
```

Comment:
can we do `from llama_hub.feishu_docs.base import FeishuDocsReader`?  it's the new default

Reply:
Yes, you can `import FeishuDocsReader from llama_hub.feishu_docs.base` as the new default.